app/views/GovukLayoutWrapper.scala.html

```diff
@@ -39,11 +38,12 @@
 
 @(
         title: PageTitle,
-        isUserResearchBannerVisible: Boolean = false,
         signOut: Boolean = true,
         timeoutActive: Boolean = true,
         showBackLink: Boolean = true,
-        showYTALink: Boolean = false
+        showYTABackLink: Boolean = false,
+        showECLBackLink: Boolean = false,
+        globalPageLayoutStyle: String = "govuk-grid-column-two-thirds"
```

Comment:
2/3 is default for all pages
need 3/4 only for summary page => this custom logic was implemented like this before